src/Microsoft.DocAsCode.YamlSerialization/Microsoft.DocAsCode.YamlSerialization.csproj

```diff
@@ -9,4 +9,8 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="TEMP\" />
+  </ItemGroup>
```

Comment:
should remove

---
In reply to: [107324542](https://github.com/dotnet/docfx/pull/1455#discussion_r107324542) [](ancestors = 107324542)